x-pack/legacy/plugins/monitoring/public/components/chart/horizontal_legend.js

```diff
@@ -61,38 +57,35 @@ export class HorizontalLegend extends React.Component {
   }
 
   createSeries(row, rowIdx) {
-    const classes = ['col-md-4 col-xs-6 monRhythmChart__legendItem'];
+    const classes = ['monRhythmChart__legendItem'];
 
     if (!includes(this.props.seriesFilter, row.id)) {
       classes.push('monRhythmChart__legendItem-isDisabled');
     }
     if (!row.label || row.legend === false) {
-      return (
-        <div
-          key={rowIdx}
-          style={{ display: 'none' }}
-        />
-      );
+      return <div key={rowIdx} style={{ display: 'none' }} />;
     }
 
     return (
       <EuiKeyboardAccessible key={rowIdx}>
-        <div
-          className={classes.join(' ')}
-          onClick={event => this.props.onToggle(event, row.id)}
-        >
-          <span className="monRhythmChart__legendLabel">
-            <span
-              className="fa fa-circle monRhythmChart__legendIndicator"
-              style={{ color: row.color }}
-              aria-label={i18n.translate('xpack.monitoring.chart.horizontalLegend.toggleButtonAriaLabel', {
-                defaultMessage: 'toggle button'
-              })}
-            />
-            { ' ' + row.label + ' ' }
-          </span>
-          { this.formatter(this.props.seriesValues[row.id], row) }
-        </div>
+        <EuiFlexItem grow={false}>
+          <div className={classes.join(' ')} onClick={event => this.props.onToggle(event, row.id)}>
```

Comment:
Consider changing this to a `<button>` tag so that you inherently get the right cursor types so users know its clickable.